code/_onclick/item_attack.dm

```diff
@@ -14,6 +14,16 @@
 
 /mob/living/attackby(obj/item/I, mob/user, params)
 	user.changeNext_move(CLICK_CD_MELEE)
+
+	if((butcher_results) && (stat == DEAD))
+		var/sharpness = is_sharp(I)
```

Comment:
is_sharp only returns 0 or 1, so this is somewhat unnecessary, especially further down where you divide the time by this value (1) as it will either not be sharp or be sharp.